### PR TITLE
update to version 1.19

### PR DESCRIPTION
**Pull request by orcachillin**
Update `pack.mcmeta` to version 9

- Dismisses "This pack was made for an older version of Minecraft"
- Updated date in pack description

### CODE REVIEW

File: Slimefun Beautified/pack.mcmeta [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
    "pack":{
-      "pack_format":8,
-      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- (v1.0)"
+      "pack_format":9,
+      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e(24/10/2022 §7- &ev1.1)"
```
Comment (54M44R) [inline Slimefun Beautified/pack.mcmeta]: Planning to replace date system with Git commit IDs. Version bump is only done before pushing out a new version.
```suggestion
      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- v1.0)"
```

**Review by 54M44R — CHANGES_REQUESTED** [review]

**Comment by gurrrrrrett3** [comment]
Requested changes made

**Review by orcachillin — COMMENTED** [review]
Requested changes made

File: Slimefun Beautified/pack.mcmeta [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
    "pack":{
       "pack_format":9,
-      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e(24/10/2022 §7- &ev1.1)"
+      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e6a588bd §7- &ev1.0)"
```
Comment (54M44R) [inline Slimefun Beautified/pack.mcmeta]: Git commit ID is not from this repository, will be changed to match version bump in the recent future from this repository. Odd `&e` placed before `v1.0` . I assume an attempt at adding color? Personally I don't believe it goes well adding color to the version number.

Preview of oddity:
image

```suggestion
      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- &v1.0)"
```

**Review by 54M44R — CHANGES_REQUESTED** [review]

File: Slimefun Beautified/pack.mcmeta [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
    "pack":{
-      "pack_format":8,
-      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- (v1.0)"
+      "pack_format":9,
+      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknow §7- v1.0)"
```
Comment (54M44R) [inline Slimefun Beautified/pack.mcmeta]: Spelling error + `(` missing before `v1.0`

**Review by 54M44R — CHANGES_REQUESTED** [review]

**Review by 54M44R — APPROVED** [review]

**Comment by orcachillin** [comment]
issues due to GPG signing, see #5

**Pull request closed without merging**